eZ/Publish/Core/Search/Common/FieldNameResolver.php

```diff
@@ -101,17 +92,17 @@ protected function getFieldMap()
                         continue;
                     }
 
-                    $this->fieldTypes[$contentType->identifier][$fieldDefinition->identifier] =
+                    $fieldTypes[$contentType->identifier][$fieldDefinition->identifier] =
                         $fieldDefinition->fieldType;
                 }
             }
         }
 
-        return $this->fieldTypes;
+        return $fieldTypes;
```

Comment:
Using this property was disabled quite some time ago, since integration test setup factory was refactored not to create all services for each test (I think it was done with container changes). It cannot work correctly since it will not be cleared if ContentType is updated. We will handle this through a separate issue, see https://jira.ez.no/browse/EZP-23941.